src/openrct2/ride/tracktypes/TrackDataVerify.cpp

```diff
@@ -0,0 +1,1335 @@
+#include "../Track.h"
```

Comment:
did you mean to include the track data files?

Reply:
For some reason these files got included in this branch, they are supposed to be for the track data refactor only.